Assets/HoloToolkit/Utilities/Plugins/README.md

```diff
@@ -0,0 +1 @@
+This folder is for the Externals.zip that should alway be up to date.
```

Comment:
Everything would compile fine, lol.  This PR would create a duplicate of the External resources but in a zipped format.  Ideally this Externals folder should only live in the [HoloToolkit](https://github.com/Microsoft/HoloToolkit) and not this repo, and this repo only having the bundled external assets that gets updated automatically (pretty much what https://github.com/Microsoft/HoloToolkit-Unity/issues/279 talks about).  Currently it would have to be done by hand, but to be honest, the main HoloToolkit doesn't change all that often.

I'm thinking about this repo in three ways:

1. This project is for learning and sharing with the community.
2. This project is for staging and creating the asset package that people will download either from here or the Unity Asset Store and should contain all the resources needed.
3. For people to download and tailor this project to export an asset package that fits their needs.